packages/types/src/constants/ForkliftControllerModel.v1beta1.ts

```diff
@@ -26,3 +26,5 @@ export const ForkliftControllerModelGroupVersionKind = {
   group: 'forklift.konveyor.io',
 };
 export const ForkliftControllerModelRef = 'forklift.konveyor.io~v1beta1~ForkliftController';
+
+export const CLUSTER_API_VERSION = `${ForkliftControllerModelGroupVersionKind.group}/${ForkliftControllerModelGroupVersionKind.version}`;
```

Comment:
this does not look ok,
a. why do we need it
b. this is the version of `ForkliftController` no the `cluster-api`

Reply:
that is a bug, each resource has it's own independent version, assuming `ForkliftController` API version will be the same for other resources is a BUG,

it works now because we are just starting and all the resources have the same `v1beta1` version, but as some resources bump version and other not, this will stop working.